pkgs/os-specific/linux/ultrablue-server/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+}:
+
+buildGoModule {
+  pname = "ultrablue-server";
+  version = "unstable-fosdem2023";
+
+  src = fetchFromGitHub {
+    owner = "ANSSI-FR";
+    repo = "ultrablue";
+    # Do not use a more recent
+    rev = "tags/fosdem-2023";
+    hash = "sha256-rnUbgZI+SycYCDUoSziOy+WxRFvyM3XJWJnk3+t0eb4=";
+    # rev = "6de04af6e353e38c030539c5678e5918f64be37e";
+  };
+
+  sourceRoot = "source/server";
+
+  vendorSha256 = "sha256-249LWguTHIF0HNIo8CsE/HWpAtBw4P46VPvlTARLTpw=";
+  doCheck = false;
```

Comment:
great, no tests

Reply:
fosdem-2023 tag requires a `/dev/tpm0` during tests, latest revision does not.

Anyway, I will be maintainer of this and will be working on it, if upstream can fix their issues quickly and we can use a more recent version, we will have tests.
Ultimately, I'm not sure something less than integration tests makes sense though for this kind of stuff ;).